<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>798</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NonIdetifier_D&amp;D</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Documents/brotherinvest/이미지/kshield.jpg</normaloff>
    <normalon>../../../../../Documents/brotherinvest/이미지/kshield.jpg</normalon>../../../../../Documents/brotherinvest/이미지/kshield.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_13">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="AnalysisTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>분석</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="INPUTbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>입력 데이터</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>17</number>
            </property>
            <property name="topMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>11</number>
            </property>
            <item row="0" column="0">
             <widget class="QTableView" name="INPUTtable"/>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="splitter_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTabWidget" name="DATAtab">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>401</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
             </font>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="DTtab">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>데이터 변환</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QGroupBox" name="TYPE4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="title">
                  <string>데이터 속성</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3">
                  <property name="leftMargin">
                   <number>5</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="ID">
                    <property name="text">
                     <string>식별자</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QRadioButton" name="QD">
                    <property name="text">
                     <string>준식별자</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QRadioButton" name="GI">
                    <property name="text">
                     <string>일반정보</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QRadioButton" name="SA">
                    <property name="text">
                     <string>민감정보</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QListWidget" name="typeListWidget">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="font">
                  <font>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                 </property>
                 <property name="currentRow">
                  <number>-1</number>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="TYPEtab">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="title">
              <string>데이터 타입</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTableWidget" name="typeTable">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="rowCount">
                 <number>10</number>
                </property>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string>컬럼</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>포맷</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>속성</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QGroupBox" name="PMbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
             </font>
            </property>
            <property name="title">
             <string>프라이버시 모델</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="privacyAdd">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>맑은 고딕</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>추가</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="privacyDelete">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>맑은 고딕</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>삭제</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QTableWidget" name="privacyTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="rowCount">
                <number>0</number>
               </property>
               <column>
                <property name="text">
                 <string>모델</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>적용값</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>적용컬럼</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QGroupBox" name="DCbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>맑은 고딕</family>
             </font>
            </property>
            <property name="title">
             <string>이용된 비식별 방법</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QTableWidget" name="methodTable">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <column>
                <property name="text">
                 <string>컬럼</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>비식별기법</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>상세 정보</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>변경된 값 수</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ResultTab">
       <attribute name="title">
        <string>결과 분석</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>비식별 레벨 추천</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="1" column="1">
            <widget class="QSplitter" name="splitter_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTableWidget" name="de_level_re_risk">
              <column>
               <property name="text">
                <string>비식별 레벨</string>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>유용성 검증(1~5)</string>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
              <column>
               <property name="text">
                <string>재식별 위험도</string>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </column>
             </widget>
             <widget class="MplWidget" name="inputGraph_3" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DetailTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>상세 분석</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="leftMargin">
            <number>11</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>11</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QGroupBox" name="INPUTDATAbox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>데이터 비교</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>11</number>
               </property>
               <item row="0" column="0">
                <widget class="QSplitter" name="splitter_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QTableView" name="INPUTDATAtable"/>
                 <widget class="QTableView" name="OUTPUTDATAtable"/>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QTabWidget" name="tabWidget_2">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="CorrelationGraph">
               <attribute name="title">
                <string> 상관 관계 그래프</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QSplitter" name="splitter_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <widget class="QWidget" name="horizontalLayoutWidget">
                   <layout class="QHBoxLayout" name="inputGraph1"/>
                  </widget>
                  <widget class="QWidget" name="horizontalLayoutWidget_2">
                   <layout class="QHBoxLayout" name="outputGraph1"/>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="RiskGraph">
               <attribute name="title">
                <string>재식별 위험 그래프</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QSplitter" name="splitter_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <widget class="MplWidget" name="inputGraph" native="true"/>
                  <widget class="MplWidget" name="outputGraph" native="true"/>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <underline>false</underline>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionimport_data"/>
   <addaction name="actionsave_data"/>
   <addaction name="actionNonIdentifier"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionimport_data"/>
    <addaction name="actionsave_data"/>
    <addaction name="actionEXIT"/>
   </widget>
   <addaction name="menuFILE"/>
  </widget>
  <action name="actionimport_data">
   <property name="text">
    <string>데이터 가져오기</string>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
  </action>
  <action name="actionsave_data">
   <property name="text">
    <string>데이터 내보내기</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>나가기</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>실행</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionNonIdentifier">
   <property name="text">
    <string>비식별 처리</string>
   </property>
  </action>
  <action name="actionnew_project">
   <property name="text">
    <string>new project</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
